CMakePresets.json

```diff
@@ -267,6 +267,22 @@
         "pipettes-384-simulator",
         "gripper-simulator"
       ]
+    },
+    {
```

Comment:
Oh wait this doesn't really make sense to me - this is fundamentally part of simulation right? should be in the `simulators` preset, since this builds headers that those need

Reply:
I didn't even think of this. 
This makes setup so much easier if I add the `setup` target to this.

Thank you!!!

Reply:
https://github.com/Opentrons/ot3-firmware/pull/435/commits/61f1b08b05a0d92c4a4a02a25275965f56a529d2